Sources/TuistSigning/ProvisioningProfile/ProvisioningProfile.swift

```diff
@@ -3,7 +3,7 @@ import TSCBasic
 import TuistSupport
 
 /// Model of a provisioning profile
-struct ProvisioningProfile: Equatable {
+public struct ProvisioningProfile: Equatable {
```

Comment:
This probably should be kept `internal`?

Reply:
✅